src/ledger/backend/connection.spec.ts

```diff
@@ -52,4 +55,24 @@ describe("ErdstallConnection", () => {
 
 		return depositRegistered;
 	});
+
+	it("allows withdrawing from the erdstall contract", async () => {
+		const withdrawRegistered = EventHelper.within(10000, conn, "Withdrawn");
+
+		const bal = new Balance(
+			0n, // epoch
+			bob.address, // account
+			true, // exit
+			assets,
+		);
+		const bp = await bal.sign(conn.erdstall(), testenv.tee);
+		const stages = await conn.withdraw(bp);
+		for (const stage of stages) {
+			const ctx = await stage.value;
+			const rec = await ctx.wait();
+			expect(rec.status, "withdrawing should have worked").to.equal(0x1);
+		}
+
+		return withdrawRegistered;
```

Comment:
What's mocha gonna do with this return value? Note that I don't understand/know the `EventHelper` ;)

Reply:
Mocha just waits for the `Promise<boolean>` to resolves and asserts that it is `true` and did not fail otherwise.

`withdrawRegistered` here is just a sanity check, that the `Withdraw` event was really triggered, with mocha waiting a maximum of 10 seconds for it.